### PR TITLE
Adds note in readme to blacklist user KMC2000 from the repo

### PR DESCRIPTION
**Pull request by ShadowDeath6**

### CODE REVIEW

File: code/modules/mob/living/simple_animal/bot/honkbot.dm [hunk 1]
```diff
@@ -120,9 +119,8 @@
 	name = "honk"
 	icon_state = "banana"
 	hitsound = 'sound/items/bikehorn.ogg'
-	damage = 0
-	damage_type = BURN
-	nodamage = 1
+	damage = 5
```
Comment (Super3222) [inline code/modules/mob/living/simple_animal/bot/honkbot.dm:122]: smh. this'll fuck over blobs.

Comment (ShadowDeath6) [reply code/modules/mob/living/simple_animal/bot/honkbot.dm:122]: sorry, this was a random line change when I merged master into my branch, don't worry about it

Comment (Super3222) [reply code/modules/mob/living/simple_animal/bot/honkbot.dm:122]: okay then good sir, this looks ready to merge :+1:

**Comment by Kmc2000** [comment]
Lol

**Comment by Mrcreepssss** [comment]
what is this exactly? the title looks weird and there is no description

**Comment by ShadowDeath6** [comment]
It adds a note in the readme so people in the future know to blacklist Kmc2000 from making changes to the repo

**Comment by Cruix** [comment]
Leave a comment

**Comment by Mrcreepssss** [comment]
i'm not sure what blacklist is or repo?

**Comment by ShadowDeath6** [comment]
Wtf cfuix this is a serious pr don't close it

**Comment by Super3222** [comment]
> i'm not sure what blacklist is or repo?

this is yogstations repo.
blacklist = ban

**Pull request closed without merging**